ee/tabby-ui/components/demo-banner.tsx

```diff
@@ -70,7 +70,7 @@ export function DemoBanner() {
   return (
     <div
       className={cn(
-        'flex items-center justify-between bg-primary px-4 text-primary-foreground transition-all md:px-5',
+        'flex items-center justify-between bg-primary px-4 text-primary-foreground transition-all md:px-5 sticky top-0 z-10',
```

Comment:
@wwayne Is it fine to set this to sticky positioning?

Reply:
I feel sticky might not fit here. Let me update the PR for your review later

Reply:
Updated, please take a check https://jam.dev/c/38b5581c-d36a-4279-a547-31eb1253955b

Reply:
Using this layout will keep the `<ButtonScrollToBottom />` invisible.